YunoHost/yunohost
# [enh] Be able to use postfix as a backup ("secondary") MX hosts

## The problem

I am using a yunohost server as backup mx for my personal arm server. I need to twik postfix conf, i could create a custom hook with an app, but i think this feature could help other people and is not so big so i suggest it to the core.

## Solution

Add 2 settings to allow to create a secondary MX able to resend mail during 20days. I think 20 days is enough to finish holidays and be happy, but we could discuss this delay.

## PR Status

Waiting CI (tested on ynh-dev and conf tested on a real server)

## How to test
```
yunohost settings set 'smtp.backup_mx.domains' -v "domain2.tld,domain3.tld"
yunohost settings set 'smtp.backup_mx.emails_whitelisted' -v "camille@domain2.tld,fred@domain3.tld"
cat /etc/postfix/relay_recipients
cat /etc/postfix/main.cf
```
You can check on a real server. You need to configure a MX record and stop your primary MX.
